src/typescript/frontend/src/components/BaseModal.tsx

```diff
@@ -32,7 +41,9 @@ export const BaseModal: React.FC<
         <div className="fixed inset-0 overflow-y-auto">
           <div className="flex min-h-full items-center justify-center p-4 text-center">
             <Dialog.Panel
-              className="w-full max-w-4xl transform border border-neutral-600 bg-black p-6 align-middle shadow-xl transition-all"
+              className={`w-full max-w-4xl transform border border-neutral-600 bg-black ${
+                disablePadding === true ? "" : "p-6"
```

Comment:
I'm a bit unclear about why this `disablePadding` prop needs to exist. Is there some styling on the deposit/withdraw form that can't be achieved by applying styles to the children of this component?

Reply:
yes, when each row is hovered, the entire row from edge to edge should highlight. but there is no way to extend the width of the rows to somehow have negative padding to "negate" padding form the parent.

Reply:
Oh no my suggestion would be to default to 0 padding and add a margin or an extra div with padding to the child components when you need it

Reply:
I actually made this change upstream so nothing to do in this PR